### PR TITLE
[Concept Entry] SwiftUI Fonts

### LINKED ISSUE #2048: [Concept Entry] SwiftUI Fonts
### Reason/inspiration (optional)

We would like a new concept entry for SwiftUI 

The edit should go in an existing file in docs/content/swiftui/concepts/fonts/fonts.md 

The entry should
- Explain `Font` `struct` (structure) in SwiftUI.
- Include header with section about standard fonts like `.largeTitle` (type property).
- Include an example with accompanying image (in media/ folder)

Please refer to the [entry template](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Concept Entrty for Existing Topic

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by gracekishino**
### Description

Added new folder `fonts` inside `content/swiftui/concepts/`
Added new entry file `content/swiftui/concepts/fonts/fonts.md` for SwiftUI Fonts.
Added a new image for the SwiftUI Fonts entry `swiftui-font.png`.

Closes #2048

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by SSwiniarski — APPROVED** [review]
@gracekishino I did a first review and it looks good to me.

File: content/swiftui/concepts/fonts/fonts.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+Below is a list of some of the `Font` methods used to modify and create fonts:
+
+- `bold()` applies a bold style.
+- `italic()` applies an italic style.
+- `weight()` applies the specified weight style.
+- `monospaced()` applies a monospaced font style.
+- `lowercaseSmallCaps()` converts lowercase characters to small capital characters.
+- `system(_ style:, design:, weight:)` applies a specified font style with optional design and/or weight styles.
+- `system(size:, weight:, design:)` applies a specified size with optional weight and/or design styles.
```
Comment (yangc95) [inline content/swiftui/concepts/fonts/fonts.md:40]:
```suggestion
- `.bold()` applies a bold style.
- `.italic()` applies an italic style.
- `.weight()` applies the specified weight style.
- `.monospaced()` applies a monospaced font style.
- `.lowercaseSmallCaps()` converts lowercase characters to small capital characters.
- `.system(_ style:, design:, weight:)` applies a specified font style with optional design and/or weight styles.
- `.system(size:, weight:, design:)` applies a specified size with optional weight and/or design styles.
```

File: content/swiftui/concepts/fonts/fonts.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+It is also possible to create custom fonts using the `custom()` methods.
```
Comment (yangc95) [inline content/swiftui/concepts/fonts/fonts.md:42]:
```suggestion
It is also possible to create custom fonts using the `.custom()` method.
```

File: content/swiftui/concepts/fonts/fonts.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+## Standard Fonts
+
+Below is a list of some of the standard `Font` type properties:
+
+- `largeTitle` a large, title font style.
+- `title` a title font style.
+- `headline` a headline font style.
+- `body` the default font style.
+- `footnote` a footnote font style.
+- `caption` a caption font style.
```
Comment (yangc95) [inline content/swiftui/concepts/fonts/fonts.md:28]:
```suggestion
- `.largeTitle` a large, title font style.
- `.title` a title font style.
- `.headline` a headline font style.
- `.body` the default font style.
- `.footnote` a footnote font style.
- `.caption` a caption font style.
```

File: content/swiftui/concepts/fonts/fonts.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+In the above example:
+
+- The first text view has `titleFont` applied. `titleFont` is created from the standard `Font` type property `largeTitle` with the method `lowercaseSmallCaps()` applied to it.
+- The next text view has the method `monospaced()` applied, setting the text to use a monospaced font.
+- The last text view has a created system font `footnoteFont` applied, with a `footnote` style, `serif` `design` and `bold` `weight`.
+
+This will display:
+
+![SwiftUI ViewModifier .font()](https://raw.githubusercontent.com/Codecademy/docs/main/media/swiftui-font.png)
```
Comment (yangc95) [inline content/swiftui/concepts/fonts/fonts.md:78]:
```suggestion
![SwiftUI Fonts](https://raw.githubusercontent.com/Codecademy/docs/main/media/swiftui-font.png)
```

File: content/swiftui/concepts/fonts/fonts.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+        .padding()
+    }
+}
+```
+
+In the above example:
+
+- The first text view has `titleFont` applied. `titleFont` is created from the standard `Font` type property `largeTitle` with the method `lowercaseSmallCaps()` applied to it.
+- The next text view has the method `monospaced()` applied, setting the text to use a monospaced font.
+- The last text view has a created system font `footnoteFont` applied, with a `footnote` style, `serif` `design` and `bold` `weight`.
```
Comment (yangc95) [inline content/swiftui/concepts/fonts/fonts.md:74]:
```suggestion
- The first text view has `titleFont` applied. `titleFont` is created from the standard `Font` type property, `.largeTitle`, with the method `.lowercaseSmallCaps()` applied to it.
- The next text view has the method `.monospaced()` applied, setting the text to use a monospaced font.
- The last text view has a created system font `footnoteFont` applied, with a `.footnote` style, `.serif` design, and `.bold` weight.
```

**Review by yangc95 — COMMENTED** [review]
@gracekishino reads very well and is a great starter! Thanks for doing some internal linking as well.

I think this will be ready to merge after the following suggestions are applied

**Comment by gracekishino** [comment]
@yangc95 thanks so much for reviewing! I have committed the suggestions.

**Review by yangc95 — APPROVED** [review]

**Comment by yangc95** [comment]
Yay @gracekishino ! live link: https://www.codecademy.com/resources/docs/swiftui/fonts

Screenshot 2023-05-18 at 7 31 19 PM

cc: @SSwiniarski

**Pull request merged**